### PR TITLE
Uploadmethod

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: coalib/bears/requirements/PackageRequirement.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from coala_decorators.decorators import generate_eq, generate_repr
+import os
```
Comment (tushar-rishav) [inline coalib/bears/requirements/PackageRequirement.py]: @Adrianzatreanu coala module imports should come after inbuilt modules. :stuck_out_tongue_winking_eye:

File: coalib/bears/requirements/PackageRequirement.py [hunk 2]
```diff
@@ -119,3 +120,19 @@ def multiple(cls, *args):
...
+    @classmethod
+    def upload(self):
+        """
+        An upload method which uploads a conda package containing the command
+        to install the corresponding package.
+        """
+        name = self.package
+        if not os.path.exists(name):
+            os.mkdir(name)
+        file = open(name + '/' + name + '.py', 'a')
```
Comment (tushar-rishav) [inline coalib/bears/requirements/PackageRequirement.py]: @Adrianzatreanu Can we not use `os.path.join(name, name + '.py' )` ?

File: coalib/bears/requirements/PackageRequirement.py [hunk 3]
```diff
@@ -119,3 +120,19 @@ def multiple(cls, *args):
...
+        file.write(self.manager + ' ' + self.package)
```
Comment (tushar-rishav) [inline coalib/bears/requirements/PackageRequirement.py]: Also I think the current file I/O approach can be made better. This is more pythonic and you won't have to close the file descriptor explicitly :
```python
with open(os.path.join('name','name.py'), 'a') as package_file:
    package_file.write("{manager} {package}".format(manager=self.manager, 
                                                    package=self.package))
```

File: coalib/bears/requirements/PackageRequirement.py [hunk 4]
```diff
@@ -119,3 +120,19 @@ def multiple(cls, *args):
...
+        file.close()
+        meta = open(name + '/meta.yaml', 'w')
```
Comment (tushar-rishav) [inline coalib/bears/requirements/PackageRequirement.py]: Same thing here

**Comment by Adrianzatreanu** [comment]
thx @tushar-rishav but this is going to be dropped most likely :D

**Pull request closed without merging**